### DISCUSSION TITLE
DAC codes

### DISCUSSION THREAD

**Discussion by fretchen** [openkfw/open-geodata-model#68] [Q&A] [answered]
I saw in the Excel file that there is a DAC Purpose Codes worksheet. What is this based on ? Is it correct that the column on the *DAC 5 code* is based on the published iati standards [version 2.03](https://iatistandard.org/en/iati-standard/203/codelists/sector/) ? Same on the description. Is this the description from the published standard or has it been edited ?

I assume that @Maja4Dev  might know more about this ?

**Comment by Maja4Dev** [comment, marked as answer]
Yes, this is correct. --Gesendet mit der WEB.DE Mail AppAm 14.06.24, 10:38 schrieb fretchen ***@***.***>:

   
  I saw in the Excel file that there is a DAC Purpose Codes worksheet. What is this based on ? Is it correct that the column on the DAC 5 code is based on the published iati standards version 2.03 ? Same on the description. Is this the description from the published standard or has it been edited ? 
  I assume that @Maja4Dev might know more about this ?
